### PR TITLE
update proc_using-a-git-provider-access-token.adoc

### PR DESCRIPTION
**Pull request by akurinnoy**
## What does this pull request change?

Added the note that personal access tokens can also be added/configured in the User Dashboard.

## What issues does this pull request fix or reference?

https://github.com/eclipse/che/issues/22112

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc [hunk 1]
```diff
@@ -25,6 +25,11 @@ pass:[<!-- vale RedHat.Spelling = YES -->]
 
 You can create and apply multiple access-token Secrets per Git provider. You must apply each of those Secrets in your user {orch-namespace}.
 
+[NOTE]
+====
+You can configure personal access tokens in *User Preferences* page (`pass:c,a,q[{prod-url}]/dashboard/#/user-preferences?tab=personal-access-tokens`) of your {prod-short} dashboard.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]:
```suggestion
You can configure personal access tokens on the *User Preferences* page of your {prod-short} dashboard: `pass:c,a,q[{prod-url}]/dashboard/#/user-preferences?tab=personal-access-tokens`.
```

**Review by deerskindoll — COMMENTED** [review]
check the suggestion and let me know what you think

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc (hunk 1 above)
Comment (akurinnoy) [reply modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]: Looks good, thanks!

Comment (ibuziuk) [reply modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]: @deerskindoll shouldn't we add `the` e.g. `on the  *User Preferences* page`

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc [hunk 2]
```diff
@@ -25,6 +25,11 @@ pass:[<!-- vale RedHat.Spelling = YES -->]
...
+====
+
 .Prerequisites
 
 * You have cluster administrator permissions for the cluster on which your organization's {prod-short} instance is running.
```
Comment (ibuziuk) [inline modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]: @vinokurig wondering why is it required to have cluster-admin permissions?

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc (hunk 1 above)
Comment (ibuziuk) [reply modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]: @akurinnoy I think we should move this to the beginning of the article smth. like 

 ```
OAuth for GitHub, GitLab, Bitbucket, or Microsoft Azure Repos needs to be xref:administration-guide:configuring-oauth-for-git-providers.adoc[configured by the administrator] of your organization's {prod-short} instance. If your administrator could not configure it for {prod-short} users, the workaround is for you to use a personal access token. You can configure personal access tokens on the *User Preferences* page of your {prod-short} dashboard: `pass:c,a,q[{prod-url}]/dashboard/#/user-preferences?tab=personal-access-tokens`, or apply it manually as a {kubernetes} Secret in the user's namespace.
```

**Review by ibuziuk — COMMENTED** [review]
@akurinnoy I think we should move it to the very beginning of the article

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc (hunk 1 above)
Comment (deerskindoll) [reply modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]: @ibuziuk you're absolutely right, `the page` is correct.

**Review by deerskindoll — APPROVED** [review]
lgtm

**Review by ibuziuk — APPROVED** [review]
approving, but please do not merge until https://github.com/eclipse-che/che-docs/pull/2604#discussion_r1202050761 is clarified. For me it is not clear why do we require cluster admin for PAT configuration

File: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc (hunk 2 above)
Comment (vinokurig) [reply modules/end-user-guide/pages/using-a-git-provider-access-token.adoc]: That's probably a leftover, @akurinnoy please remove the item.

**Pull request merged**